pepper-apis/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/form/block/question/QuestionCreatorHelper.java

```diff
@@ -203,7 +203,7 @@ PicklistQuestion createPicklistQuestion(AIBuilderContext ctx, PicklistQuestionDe
                 (picklistGroupDef) -> ctx.getAIBuilderFactory().getPicklistCreatorHelper()
                         .createPicklistGroup(ctx, picklistGroupDef));
 
-        List<PicklistOption> picklistOptions = CollectionMiscUtil.createListFromAnotherList(questionDef.getAllPicklistOptions(),
+        List<PicklistOption> picklistOptions = CollectionMiscUtil.createListFromAnotherList(questionDef.getPicklistOptions(),
```

Comment:
getAllPicklistOptions() includes nestedPicklistOptions as main options. 
looks like getPicklistOptions() should be the correct one to use here. 
looks good in front-end and backend response.
Need to check if it's fine for csv export and elastic export

Reply:
--
Would have been better to create Group options in PicklistCreatorHelper.createPicklistGroup() . We can set groupId from here right away.